backend/code/archipelag/market/views.py

```diff
@@ -4,10 +4,11 @@
 from django.contrib.auth.decorators import login_required
 from json import loads
 from archipelag.market.models import Market
```

Comment:
Do you use rule, in which you divide imports into 3 groups:
- from python STD
- external packages
- internal packages

?
If yes, please fix this.

Reply:
ok I will use this rule in this project